src/test/integration/fixtures/build-modules/expected/bundled-amd-minified/index.html

```diff
@@ -0,0 +1 @@
uire(e,r,i)},req.config=function(e){return req(e)},req.nextTick="undefined"===typeof setTimeout?function(e){e()}:function(e){setTimeout(e,4)},require||(require=req),req.version="2.3.5",req.jsExtRegExp=/^\/|:|\?|\.js$/,req.isBrowser=isBrowser,s=req.s={contexts:contexts,newContext:newContext},req({}),each(["toUrl","undef","defined","specified"],function(e){req[e]=function(){var r=contexts._;return r.require[e].apply(r,arguments)}}),isBrowser&&(head=s.head=document.getElementsByTagName("head")[0],baseElement=document.getElementsByTagName("base")[0],baseElement&&(head=s.head=baseElement.parentNode)),req.onError=defaultOnError,req.createNode=function(e){var r=e.xhtml?document.createElementNS("http://www.w3.org/1999/xhtml","html:script"):document.createElement("script");return r.type=e.scriptType||"text/javascript",r.charset="utf-8",r.async=!0,r},req.load=function(r,i,n){var e,t=r&&r.config||{};if(isBrowser)return e=req.createNode(t,i,n),e.setAttribute("data-requirecontext",r.contextName),e.setAttribute("data-requiremodule",i),!e.attachEvent||e.attachEvent.toString&&0>e.attachEvent.toString().indexOf("[native code")||isOpera?(e.addEventListener("load",r.onScriptLoad,!1),e.addEventListener("error",r.onScriptError,!1)):(useInteractive=!0,e.attachEvent("onreadystatechange",r.onScriptLoad)),e.src=n,t.onNodeCreated&&t.onNodeCreated(e,t,i,n),currentlyAddingScript=e,baseElement?head.insertBefore(e,baseElement):head.appendChild(e),currentlyAddingScript=null,e;if(isWebWorker)try{setTimeout(function(){},0),importScripts(n),r.completeLoad(i)}catch(t){r.onError(makeError("importscripts","importScripts failed for "+i+" at "+n,t,[i]))}},isBrowser&&!cfg.skipDataMain&&eachReverse(scripts(),function(e){if(head||(head=e.parentNode),dataMain=e.getAttribute("data-main"),dataMain)return mainScript=dataMain,cfg.baseUrl||-1!==mainScript.indexOf("!")||(src=mainScript.split("/"),mainScript=src.pop(),subPath=src.length?src.join("/")+"/":"./",cfg.baseUrl=subPath),mainScript=mainScript.replace(jsSuffixRegExp,""),req.jsExtRegExp.test(mainScript)&&(mainScript=dataMain),cfg.deps=cfg.deps?cfg.deps.concat(mainScript):[mainScript],!0}),define=function(e,r,i){var n,t;"string"!==typeof e&&(i=r,r=e,e=null),isArray(r)||(i=r,r=null),!r&&isFunction(i)&&(r=[],i.length&&(i.toString().replace(commentRegExp,commentReplace).replace(cjsRequireRegExp,function(e,i){r.push(i)}),r=(1===i.length?["require"]:["require","exports","module"]).concat(r))),useInteractive&&(n=currentlyAddingScript||getInteractiveScript(),n&&(!e&&(e=n.getAttribute("data-requiremodule")),t=contexts[n.getAttribute("data-requirecontext")])),t?(t.defQueue.push([e,r,i]),t.defQueueMap[e]=!0):globalDefQueue.push([e,r,i])},define.amd={jQuery:!0},req.exec=function(text){return eval(text)},req(cfg)}})(this,"undefined"===typeof setTimeout?void 0:setTimeout);</script><script>define('polymer-build-generated-module-0', ['shared_bundle_1.js']);</script><script>require(['polymer-build-generated-module-0']);</script></head><body></body></html>
```

Comment:
Can we use the code from the build test infra to replace common scripts with comments? This is super hard to read/review without it

Reply:
I would like to, agree these are very hard to read. It's a little more work than I originally thought when we talked, because it makes re-generating the goldens more complicated (which I am doing by just dumping out the whole dir), since I will need to transform the files in the temp dir or copy them one by one, rather than just copying the dirs over wholesale. Not a huge deal, but ok if I file an issue for now?

(I also removed this particular new test case anyway, because it wasn't actually testing anything already covered by other ones.)